<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VPageView</class>
 <widget class="QWidget" name="VPageView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>2026</width>
    <height>94</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>1763</width>
     <height>82</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="sizeConstraint">
     <enum>QLayout::SetDefaultConstraint</enum>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="HPageButton" name="btnAuto">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>24</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">border:none</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="Images.qrc">
        <normaloff>:/Images/Images/Button2.png</normaloff>
        <normalon>:/Images/Images/Button1.png</normalon>:/Images/Images/Button2.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>210</width>
        <height>80</height>
       </size>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="HPageButton" name="btnProduct">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>24</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">border:none</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="Images.qrc">
        <normaloff>:/Images/Images/Button2.png</normaloff>
        <normalon>:/Images/Images/Button1.png</normalon>:/Images/Images/Button2.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>210</width>
        <height>80</height>
       </size>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="HPageButton" name="btnManual">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>24</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">border:none</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="Images.qrc">
        <normaloff>:/Images/Images/Button2.png</normaloff>
        <normalon>:/Images/Images/Button1.png</normalon>:/Images/Images/Button2.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>210</width>
        <height>80</height>
       </size>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="HPageButton" name="btnWork">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>24</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">border:none</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="Images.qrc">
        <normaloff>:/Images/Images/Button2.png</normaloff>
        <normalon>:/Images/Images/Button1.png</normalon>:/Images/Images/Button2.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>210</width>
        <height>80</height>
       </size>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="HPageButton" name="btnMachine">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>24</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">border:none</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="Images.qrc">
        <normaloff>:/Images/Images/Button2.png</normaloff>
        <normalon>:/Images/Images/Button1.png</normalon>:/Images/Images/Button2.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>210</width>
        <height>80</height>
       </size>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="HPageButton" name="btnVision">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>24</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">border:none</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="Images.qrc">
        <normaloff>:/Images/Images/Button2.png</normaloff>
        <normalon>:/Images/Images/Button1.png</normalon>:/Images/Images/Button2.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>210</width>
        <height>80</height>
       </size>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="HPageButton" name="btnMaintain">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>24</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">border:none</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="Images.qrc">
        <normaloff>:/Images/Images/Button2.png</normaloff>
        <normalon>:/Images/Images/Button1.png</normalon>:/Images/Images/Button2.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>210</width>
        <height>80</height>
       </size>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="HPageButton" name="btnAlarm">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>24</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">border:none</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="Images.qrc">
        <normaloff>:/Images/Images/Button2.png</normaloff>
        <normalon>:/Images/Images/Button1.png</normalon>:/Images/Images/Button2.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>210</width>
        <height>80</height>
       </size>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>HPageButton</class>
   <extends>QPushButton</extends>
   <header>hpagebutton.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="Images.qrc"/>
 </resources>
 <connections/>
</ui>
